packages/blocks/src/page-block/default/utils.ts

```diff
@@ -108,15 +108,23 @@ function isPointIn(block: DOMRect, x: number, y: number): boolean {
 }
 
 export async function downloadImage(model: BaseBlockModel) {
-  const link = document.createElement('a');
-  const url = await getUrlByModel(model);
-  url && (link.href = url);
-  link.setAttribute('target', '_blank');
-  document.body.appendChild(link);
-  link.download = 'test';
-  link.click();
-  document.body.removeChild(link);
-  link.remove();
+  const imgSrc = await getUrlByModel(model);
+  const image = new Image();
+  imgSrc && (image.src = imgSrc);
+  image.setAttribute('crossOrigin', 'anonymous');
+  image.onload = function () {
+    const canvas = document.createElement('canvas');
+    canvas.width = image.width;
+    canvas.height = image.height;
+    const context = canvas.getContext('2d');
+    context && context.drawImage(image, 0, 0, image.width, image.height);
+    const url = canvas.toDataURL('image/png');
```

Comment:
Images are blobs with our blob storage, why not convert the blob and export it using `File`? The canvas approach will erase all the metadata of the image, IMO that doesn't sound acceptable. See https://stackoverflow.com/questions/27159179/how-to-convert-blob-to-file-in-javascript

Reply:
Because the file type was text when I  used blob download from storage.